src/scss/base.scss

```diff
@@ -1,15 +1,16 @@
+
 @include oFontsInclude(BentonSans, normal);
 
-.o-ft-header {
+.o-header {
 	// 80 px for the primary container and 5 for an empty secondary or 25 for a secondary with content
 	min-height: 85px;
 	max-height: 125px;
 	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
```

Comment:
This is perhaps a comment for o-fonts but I'll raise here... I don't think modules should have references to actual font families and instead refer to a 'use case'. e.g. something like:
````
font-family: oFontGetFamily(body);
````